test/e2e/s3/cross-account.sh

```diff
@@ -23,6 +29,32 @@ debug_msg "executing test: $service_name/$test_name"
 bucket_name="ack-test-smoke-$service_name-$AWS_ACCOUNT_ID_ALT-$RANDOM"
 resource_name="buckets/$bucket_name"
 
+# Create the testing namespace.
+# Note: this is step is not necessary.
```

Comment:
I would rather that the namespace under test here be created and deleted by the test itself. In other words, let's make the test entirely idempotent and self-isolated. Create a randomly-named Namespace automatically at the start of the test and delete it at the end.

Reply:
++ Done.